src/FunctionApps/python/IntakePipeline/__init__.py

```diff
@@ -27,20 +31,65 @@ def run_pipeline():
     )
 
     container_url = get_required_config("INTAKE_CONTAINER_URL")
-    container_prefix = get_required_config("INTAKE_CONTAINER_PREFIX")
-    output_path = get_required_config("OUTPUT_CONTAINER_PATH")
+    valid_output_path = get_required_config("VALID_OUTPUT_CONTAINER_PATH")
```

Comment:
This is a lot to put in the major orchestrating function - can we refactor a little?